### ISSUE TITLE
lz4: build fails

### ISSUE THREAD

**Issue by waddlesplash** [haikuports/haikuports#1519]
```
Warning: Error: found provides for "haiku >= r1~alpha1", but none matching the version requirement
```
http://vmpkg.haiku-os.org/kitchen/#/build/29

**Comment by pulkomandy** [comment]
Just remove the >= $haikuVersion from the recipe?

**Comment by korli** [comment]
Hopefully fixed in 3153964c03b51e126803dda49eaf6155bfcf490b
